### ISSUE TITLE
select_performance, star not reported

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1942]
```abap
SELECT * FROM tab INTO @DATA(ls_tab) WHERE field = @l_var ORDER BY blah.
ENDSELECT.
```
